packages/builder/src/components/backend/DatasourceNavigator/DatasourceNavigator.svelte

```diff
@@ -171,10 +171,7 @@
     }
 
     // Check for a matching viewV2
-    const selectedViewV2 = viewsV2.selected?.name
-    const viewV2Table = options.find(table => {
-      return table.views?.[selectedViewV2] != null
-    })
+    const viewV2Table = options.find(x => x._id === viewsV2.selected?.tableId)
```

Comment:
Just to double check, is this meant to be tableId or view.id?

Reply:
Table - this code is trying to check if a certain datasource (e.g. a postgres datasource) contains the active entity (which could be a table or view or anything). This particular line is trying to find a table in that datasource that contains the selected view. So we're trying to find a table from `options` (which is the list of tables in this datasource) which has the ID of the selected view table ID. If we find a table, it means we contain the active entity.